### ISSUE TITLE
Style of data names

### ISSUE THREAD

**Issue by nautolycus** [COMCIFS/cif_ed#5]
The data names in the first draft of the dictionary have been suggested by the NanED standardisation committee and members of the IUCr journals editorial office. While CIF data names are technically 'meaningless' (all attributes should be declared explicitly in the CIF dictionary definitions), well-chosen terms are appreciated by the community.

At this time, any portions of new data names following the full point may be changed for more appropriate terminology (the portion before the full point is determined by the category to which the item belongs). However, some specific proposals have been advanced, based on usage in other CIF dictionaries:

 * `_diffrn_measurement.tracking`: Items with the "yes/no" values in the CIF_CORE dictionary often have the "_flag" postfix, so renaming the item to `_diffrn_measurement.tracking_flag` or `_diffrn_measurement.tracking_method_flag` might be clearer?

 * `_diffrn_measurement.method_precession`: similarly, `_diffrn_measurement.precession_method_flag` or `_diffrn_measurement.precession_flag` ?

 * `_diffrn_measurement.integration`: Since the values are drawn from an enumerated list, maybe the item should be renamed to something like `_diffrn_measurement.integration_mode`, `_diffrn_measurement.integration_type` or `_diffrn_measurement.integration_method`?
 
Community feedback on these and other data names welcome at this stage.

**Comment by vaitkus** [comment]
The post-community review PR (https://github.com/COMCIFS/cif_ed/pull/9) made changes to some of the data name in question. Specifically:

*  `_diffrn_measurement.tracking` was renamed to `_diffrn_measurement.sample_tracking` .
*  `_diffrn_measurement.integration` was renamed `_diffrn_measurement.rotation_mode`.

I guess the suggestion to add the `_flag` postfix to yes/no valued data items was not accepted.

**Comment by nautolycus** [comment]
Yes, the submitted PR reflects the conclusions of the NanED/Commission on Electron Crystallography consultations.
